client/src/components/TransactionReports.jsx

```diff
@@ -258,6 +254,12 @@ class TransactionReports extends React.Component {
   componentDidMount() {
     this.getIpnData();
     this.getIpnCount();
+    this.timer = setInterval(()=> this.getIpnData(), 5000);
```

Comment:
Ideally server.js should send updates instead of pegging the server a billion times.